src/math_util.rb

```diff
@@ -0,0 +1,43 @@
+class MathUtil
+
+  def self.sum_even_numbers(numbers)
+    numbers&.select(&:even?)&.sum
+  end
+
+  def self.two_largest_numbers(numbers)
+    return nil if numbers.nil? ||  numbers.size < 2
```

Comment:
Same as before, how can you remove the checks for `nil`?

Reply:
I imagine you'd use it like:
`firstNumber, secondNumber = MathUtil.two_largest_numbers(arrayOfNumbers)`

How do you use it when it returns `nil`?

Reply:
Given that you chose to do validations (even though it was not specifically asked for in the problem), why did you chose to return `nil` in case the input is invalid instead of raising an error?
This applies for all three problems

Reply:
I use it like:
```
result = MathUtil.two_largest_numbers(arrayOfNumbers)
```
where result is an array.

Also, I read this post "<i>Avoiding “NoMethodError” for nil in Ruby</i>": https://dlet.me/avoiding-nomethoderror-for-nil-in-ruby-9ce0f3fd0e30